packages/select/src/vaadin-select.js

```diff
@@ -296,7 +296,7 @@ class Select extends DelegateFocusMixin(
     super();
 
     // Ensure every instance has unique ID
-    const uniqueId = (Select._uniqueId = 1 + Select._uniqueId || 0);
+    const uniqueId = (Select._uniqueSelectId = 1 + Select._uniqueSelectId || 0);
```

Comment:
Seems that there are no corresponding tests on the `id` attribute for the select component.

Reply:
Thanks, added.